exercises/hello-world/example.cpp

```diff
@@ -0,0 +1,13 @@
+#include "hello_world.h"
+
+using namespace std;
+
+namespace hello_world
+{
+	
+std::string hello()
+{
+	return "Hello, World!";
```

Comment:
I think this is another tab, please change it to 4 spaces so the repo stays consisted :)